index.html

```diff
@@ -234,6 +226,35 @@ <h3>BrowserCaptureMediaStreamTrack extension</h3>
         </dl>
       </section>
     </section>
+    <section id="applying-the-restriction-transformation">
+      <h2><dfn>Applying the restriction transformation</dfn></h2>
+      <p>
+        When it's time to produce <var>frame</var> for a video track <var>videoTrack</var> that is
+        [=restricted=] to a given target <var>cropTarget</var>, execute the following algorithm:
+      </p>
+      <ol>
+        <li>
+          <!-- TODO(eladalon1983): Fix linkability of slots in the mediacapture-region spec. -->
+          If <var>cropTarget</var>.[[\Element]] does not represent an element that's in the DOM of
+          either the captured [=top-level browsing context=] or one of its descendant [=browsing
+          contexts=], abort without producing a new frame.
+        </li>
+        <li>
```

Comment:
I think we want to say, after the step that clips to the intersection with the viewport, that if the result consists of zero pixels -> abort without producing a new frame. The formulation of "zero pixels" would allow for other reasons than viewport-intersection, provided they have a similar result. Wdyt?

Reply:
Resolving because I've added an item that skips the frame if the intersection with the viewport is empty. (Please unresolve if you think this merits further discussion.)

About some other topics touched upon in this comment threads:
* Fully occluded content should still be captured if its bounding box's intersection with the viewport is non-empty.
* I think fragmented elements can theoretically make for valid targets, though it's OK if implementations don't implement it right away.
* I'll add something separate to cover the backdrop root issue. (Or does either of these fit?)

Reply:
It looks like the box fragment and backdrop root restrictions made it into the final PR.  LGTM.